docs/plugin_information.asciidoc

```diff
@@ -0,0 +1,16 @@
+:default_plugin: 0
+
+//////////////////////////////////////////
+GENERATED FILE DO NOT EDIT
+//////////////////////////////////////////
+* Version: %\{VERSION\}
+* Released on: %\{RELEASE_DATE\}
+* %\{CHANGELOG_URL\}[Changelog]
+
+ifeval::[{default_plugin} == 0]
+
+==== Installation
+
+For plugins not bundled by default, it is easy to install by running +bin/logstash-plugin install logstash-{type}-{plugin}+. See <<working-with-plugins>> for more details.
+
+endif::[]
```

Comment:
This part will be generated from the https://github.com/elastic/docs-tools/ run, the default plugin section need to be there, I don't think we want people to be able to mess with that.